modules/maps.js

```diff
@@ -477,8 +463,11 @@ class MapCrafter {
                 this.maybeUpdateFragmentsNeeded(currentMap);
                 while (!this.canAfford() && level > this.profile.minLevel) {
                     level -= 1;
-                    if (this.profile.haveMapReducer && level === this.profile.z && level > this.profile.minLevel) {
-                        level -= 1; // if we have map reducer, skip the zone level
+                    if (this.profile.haveMapReducer
+                        && level === this.profile.z
+                        && level > this.profile.minLevel
+                        && !mapLevelHasPrestiges(level)) {
+                        level -= 1; // if we have map reducer, and there are no prestiges in the map, skip that level
                     }
```

Comment:
Same thing here.
Sorry for bothering you about this by the way.

Reply:
Done:)